.tekton/release-service-push.yaml

```diff
@@ -223,6 +246,48 @@ spec:
         operator: in
         values:
         - "true"
+    - name: build-sealights-container
+      params:
+      - name: IMAGE
+        value: $(params.output-image).sealights
+      - name: DOCKERFILE
+        value: $(params.dockerfile)
+      - name: CONTEXT
+        value: $(params.path-context)
+      - name: HERMETIC
+        value: $(params.hermetic)
+      - name: PREFETCH_INPUT
+        value: $(params.prefetch-input)
+      - name: IMAGE_EXPIRES_AFTER
+        value: $(params.image-expires-after)
+      - name: COMMIT_SHA
+        value: $(tasks.clone-repository.results.commit)
+      - name: BUILD_ARGS
+        value:
+        - $(params.build-args[*])
+      - name: BUILD_ARGS_FILE
+        value: $(params.build-args-file)
+      - name: SOURCE_ARTIFACT
+        value: $(tasks.sealights-go-instrumentation.results.SOURCE_ARTIFACT)
+      - name: CACHI2_ARTIFACT
+        value: $(tasks.prefetch-dependencies.results.CACHI2_ARTIFACT)
+      runAfter:
+      - sealights-go-instrumentation
+      - prefetch-dependencies
```

Comment:
I think this Task now doesn't depend on `prefetch-dependencies` Task in "push" PipelineRun

Reply:
Should depends on preferch-dependencies due that is using a result from there tasks.prefetch-dependencies.results.CACHI2_ARTIFACT